Jordan Massingill

jordan-massingill opened the pull request:
Initial PR
Created React App

jordan-massingill committed:
Initial PR

Initial PR
Created React App

jordan-massingill committed:
MVP complete

Completed MVP
Working on add friend functionality

jordan-massingill committed:
Add Friend Functionality working

Completed add friend functionality
need to add the update and delete functions

groov1234 commented on friends/src/App.js:
```diff
@@ -49,24 +49,24 @@ class App extends Component {
     .then(result => {
       this.setState({
       friendName: '',
-      friendAge: undefined,
```
If you want to leave something undefined. `null`. However, `' '`, is more suited for this use case as far as I know. Well done!

groov1234 replied:
Correction. Further research seems to conclude that `null` would be a more proper value for the initial state definition of `friendAge`

jordan-massingill replied:
I tried using null, but it wouldn't let me. I'll try again!

groov1234 commented on friends/src/App.js:
```diff
@@ -49,24 +49,24 @@ class App extends Component {
   render() {
     return (
       <div className="App">
+        <Route path='/' render={props => <FriendForm {...props} newFriendName={this.state.friendName} newFriendAge={this.state.friendAge} newFriendEmail={this.state.friendEmail} nameAdd={this.addNameHandler} ageAdd={this.addAgeHandler} emailAdd={this.addEmailHandler} submitFriend={this.addNewFriendHandler} />} />
```
Looks as though this one gave you some trouble. A quick tip. Start a new line after each JSX attribute / Props combination. You'll find that it is more readable and easier to spot typos and other varying issues.

jordan-massingill committed:
Initial PR

Initial PR
Added FriendCard Component
Working on functionality of updating individual friends

jordan-massingill committed:
Completed Edit functionality

jordan-massingill committed:
Minor Changes

jordan-massingill committed:
Completed edit and delete functionality
